### ISSUE TITLE
🐞 Bug mixed framework project creation is possible in SPFx generator 1.7.0

### ISSUE THREAD

**Issue by StfBauer** [SharePoint/sp-dev-docs#2967]
#### Category
- [x] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior
Choose a framework for Application Customizer to make use of ReactJS, KnockoutJS of any other framework.
Chose framework only once on project creation.

#### Observed Behavior
Honestly, I am not sure if it was always like this or has just changed recently. When I create a new project of type ApplicationCustomizer the user does not get asked for the framework to use.

![screenshot 2018-11-18 at 11 31 40](https://user-images.githubusercontent.com/5503835/48671268-db474b00-eb25-11e8-88b0-398edfc2b994.png)

In case a ReactJS control or Knockoutjs should get loaded in an extension a user need to add the references later manually to make use of it. 

When a web part will be added later to the solution the selection for the framework pop up.

![screenshot 2018-11-18 at 11 38 37](https://user-images.githubusercontent.com/5503835/48671349-d8008f00-eb26-11e8-8b92-7368b6167bce.png)

This framework selection never gets stored to the `.yo-rc.json`. Causing the next web part again askes for the framework selection again.

![screenshot 2018-11-18 at 11 52 02](https://user-images.githubusercontent.com/5503835/48671451-baccc000-eb28-11e8-9669-f579686e8845.png)

The result is that this then results in a mixed project of Knockout, ReactJS, noframework.

<img width="416" alt="screenshot 2018-11-18 at 11 56 04" src="https://user-images.githubusercontent.com/5503835/48671473-18f9a300-eb29-11e8-9a6f-d27736dd0c92.png">

From an SPFx generator perspective, there are templates available for KnockoutJS, Reactjs, and no framework.

<img width="543" alt="screenshot 2018-11-18 at 11 44 30" src="https://user-images.githubusercontent.com/5503835/48671395-97554580-eb27-11e8-9c43-09fdee24c76a.png">

These templates are not existent in SharePoint Framework 2019 Application Customizer generator.

<img width="431" alt="screenshot 2018-11-18 at 11 48 27" src="https://user-images.githubusercontent.com/5503835/48671412-eef3b100-eb27-11e8-9709-9ee8744177f5.png">

#### Steps to Reproduce

Create a new project using SPFx generator 1.7.0:
1)  create a solution using ApplicationCustomizer
2) add web part using ReactJS Framework
3) add web part using KnockoutJS Framework

In this case every time the framework selection dialogue pop-up.

The same problem exists when the following solution gets created:
1) add web part using ReactJS Framework
2) add web part using KnockoutJS Framework

Even in this case every time the framework selection dialogue pop-up.

**Comment by patmill** [comment]
Yup, it looks like that is the case.  However, given the minimal impact, we're going to mark this as won't fix, but will keep it in mind if we do more fundamental changes in the generator.
